addons/web/static/src/core/utils/numbers.js

```diff
@@ -32,3 +32,26 @@ export function computeVariation(value, comparisonValue) {
 export function clamp(num, min, max) {
     return Math.max(Math.min(num, max), min);
 }
+
+/**
+ * A function to create flexibly-numbered lists of integers, handy for each and map loops.
+ * start, if omitted, defaults to 0;
+ * step defaults to 1 if start is before stop, otherwise -1.
+ * Returns a list of integers from start (inclusive) to stop (exclusive), incremented (or decremented) by step.
+ * @param {number} min default 0
+ * @param {number} max
+ * @param {number} step default 1
+ * @returns {number[]}
+ */
+export function range(start = 0, stop, step = 1) {
+    const array = [];
+    if (!stop) {
+        stop = start;
+        start = 0;
+    }
```

Comment:
Better if there is only 1 signature `range(start, stop, step = 1)`

Reply:
ok